kafkaesk/consumer.py

```diff
@@ -300,6 +305,8 @@ async def _consume_batch(
         # Look for failures
         for task in done:
             record = futures[task]
+            tp = aiokafka.TopicPartition(record.topic, record.partition)
+            self._tp[tp] = record.offset + 1
```

Comment:
These are ensured they will return in order right? You don't need to do a `max` of existing value to current one being placed?

Reply:
good catch!